lib/gooddata/lcm/actions/synchronize_date_dimension.rb

```diff
@@ -0,0 +1,128 @@
+# (C) 2019-2020 GoodData Corporation
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class SynchronizeDateDimension < BaseAction
+      DESCRIPTION = 'Synchronize Date Dimension'
+      DATE_DIMENSION_CUSTOM_V2 = 'custom_v2'
+      DATE_DIMENSION_OLD = %w[gooddata custom]
```

Comment:
please define full urn: urn:gooddata:date or urn:custom:date to urn:custom_v2:date